scripts/standalone.mk

```diff
@@ -32,6 +32,18 @@ ifeq ($(RISCV_CMODEL),)
 RISCV_CMODEL = medany
 endif
 
+ifeq ($(SPEC),)
+SPEC=nano
+endif
+
+ifeq ($(SPEC),nano)
+LIBMETAL_EXTRA=-lmetal-gloss
+endif
+
+ifeq ($(SPEC),picolibc)
+LIBMETAL_EXTRA=-lmetal-pico
+endif
```

Comment:
Using SPEC internally works, but I'd prefer to have a more descriptive option on the frontend for users to actually choose their libc implementation like `RISCV_LIBC` or similar

Reply:
Or just LIBC? And add some error checking so if the value is set to something crazy, it exits instead of breaks.

Reply:
And I added error checking so you get a message from make when you use an invalid libc value. I haven't added any documentation about the supported values yet.